roomserver/storage/postgres/deltas/20221027084407_published_appservice.go

```diff
@@ -29,6 +29,13 @@ func UpPulishedAppservice(ctx context.Context, tx *sql.Tx) error {
 	if err != nil {
 		return fmt.Errorf("failed to execute upgrade: %w", err)
 	}
+	_, err = tx.ExecContext(ctx, `
+		ALTER TABLE roomserver_published DROP CONSTRAINT IF EXISTS roomserver_published_pkey;
```

Comment:
@Danieloni1 looks like a dendrite bug defining the wrong one?
Should we also open an issue about it in official dendrite?

Reply:
No, I think these are residues from the first time we initialized the `roomserver_published` table schema.
They have fixed this a long time ago but we were stuck with the wrong table initialization from before the fix.
Look: https://github.com/matrix-org/dendrite/blob/main/roomserver/storage/postgres/published_table.go#L38

Reply:
Just to be clear:
We had `PRIMARY KEY (room_id)` in our DB when I manually checked the `roomserver_published` table specs.